lattices/LEL/test/tLEL.cc

```diff
@@ -316,7 +316,7 @@ int main (int argc, const char* argv[])
 //
    cout << endl << "LELUnary<Float>" << endl;
   {
-    CountedPtr<LELInterface<Float> > pExpr = new LELLattice<Float>(bF);
+    std::shared_ptr<LELInterface<Float> > pExpr(new LELLattice<Float>(bF));
```

Comment:
`std::make_shared` avoids the duplicate `LELInterface<Float>` type here. There are many other similar occurences in this file and other files below, that can be shortened this way.
```suggestion
   auto pExpr = std::make_shared<LELLattice<Float>>(bF);
```

Reply:
Now using make_shared in lattices test programs and in a few other source files.